docs/docs/cli-commands.md

```diff
@@ -1074,10 +1074,16 @@ export const schema = gql`
     email: String
     name: String
   }
+
+  type Mutation {
+    createUser(input: CreateUserInput!): User! @requireAuth
+    updateUser(id: Int!, input: UpdateUserInput!): User! @requireAuth
+    deleteUser(id: Int!): User! @requireAuth
+  }
 `
 ```
 
-The services file fulfills the query. If the `--crud` option is added, this file will be much more complex.
+The services file fulfills the query with . If the `--crud=false` option is added, this file will be less complex.
```

Comment:
```suggestion
The services file fulfills the query. If the `--no-crud` option is added, this file will be less complex.
```

You added ` with ` to the first sentence here. I don't understand how to read that to make any sense out of it. The extra space after "with " makes me think you forgot some word, or had a thought you didn't finish.